magento/ece-tools
# MAGECLOUD-3345: Add Node.JS Docker Image

### Fixed Issues (if relevant)

[MAGECLOUD-3345](https://magento2.atlassian.net/browse/MAGECLOUD-3345)

### Manual testing scenarios

1. Run `php vendor/bin/ece-tools docker:build --node=8`
1. Verify `node` container has been added to `docker-compose.yml` with version `8`

### Contribution checklist
